daemon/lokinet-vpn

```diff
@@ -1,59 +1,141 @@
 #!/usr/bin/env python3
```

Comment:
This basic working example script is fine, but I think we should have a separate document describing input and output from the LMQ endpoint.  (Can come in a later PR).